scripts/run_simulator.py

```diff
@@ -18,11 +19,22 @@
 
 if (Path(__file__).parent / 'simulator/VERSION').exists():
     print("Running in release mode")
-    SIM_BASE = Path(__file__).parent
+    SIM_BASE = Path(__file__).parent.resolve()
 else:
     print("Running in development mode")
     # Assume the script is in the scripts directory
-    SIM_BASE = Path(__file__).parents[1]
+    SIM_BASE = Path(__file__).parents[1].resolve()
+
+POSSIBLE_WEBOTS_PATHS = [
+    ("darwin", "/Applications/Webots.app/Contents/MacOS/webots"),
+    ("win32", "C:\\Program Files\\Webots\\msys64\\mingw64\\bin\\webotsw.exe"),
+    ("win32", expandvars("%LOCALAPPDATA%\\Programs\\Webots\\msys64\\mingw64\\bin\\webotsw.exe")),
+    # Attempt to use the start menu shortcut
+    ("win32", expandvars("%ProgramData%\\Microsoft\\Windows\\Start Menu\\Programs\\Cyberbotics\\Webots.lnk")),
```

Comment:
Nothing wrong here  - just commenting to say this is interesting, I didn't know this was how you handled environment vars in Python... neat.